pages/map.js

```diff
@@ -194,11 +194,9 @@ export default function Map({ data }) {
                 />
               ))}
         </div>
-        <div className="h-screen">
           <DynamicMap
```

Comment:
This div wrapper needs to be here.
It prevents the layout from shifting while the map is loading.
It should be
```js
<div style={{ height: 'min(96vw, 100vh)' }}>
```

Reply:
I have updated it.